downsampling/generated_manifest.yml

```diff
@@ -0,0 +1,19 @@
+# This is a file generated by the scripts/generate script
```

Comment:
renaming the file to something that isn't manifest may help disambiguate this manifest from the template manifest. Since our namign conventions still are not shared, it seems likely to happen to users as well.  Perhaps `meta.yml`?

Reply:
works for me - I'm not super opinionated on name, just so long as we make it clear it's a machine generated file

Reply:
is the root lvl looking for a specific file name? The fact that it is generated or not can be left to a comment at the head of the file like you have. The file name itself, likely doesn't need to include `generated`. Especially if users end up copy pastaing to pay dirt, which I am a fan of as you might have noticed lol